jung-algorithms/src/test/java/edu/uci/ics/jung/layout/spatial/FRLayoutsTimingTest.java

```diff
@@ -31,7 +31,7 @@
 
   private static final Logger log = LoggerFactory.getLogger(FRLayoutsTimingTest.class);
   Graph<String> graph;
-  LayoutModel<String> layoutModel;
+  AbstractLayoutModel<String> layoutModel;
 
```

Comment:
I did not want to put another getter into the interface, so I am declaring as the abstract class

Reply:
(see comment above about the getTheFuture() method)